README.md

```diff
@@ -183,6 +183,7 @@ Provisioning.PasswordCryptSaltLength=10
 ResourceDisk.Format=y
 ResourceDisk.Filesystem=ext4
 ResourceDisk.MountPoint=/mnt/resource
+ResourceDisk.MountOptions=None
```

Comment:
azurelinuxagent/daemon/resourcedisk/default.py
line 147 `if mount_options is not None:`